stdlib/src/collections/dict.mojo

```diff
@@ -799,6 +799,23 @@ struct Dict[K: KeyElement, V: CollectionElement](
             return default.value()[]
         raise "KeyError"
 
+    fn popitem(inout self) raises -> (K, V):
+        """Remove and return an arbitrary key-value pair from the dictionary.
+
+        Returns:
+            A tuple containing the key and value of the removed item.
+
+        Raises:
+            "KeyError" if the dictionary is empty.
+        """
+        if self.size == 0:
+            raise "KeyError: popitem(): dictionary is empty"
+        var entry = self._entries.pop(0)
```

Comment:
Shouldn't we follow the default behaviour like python in LIFO order?

Reply:
You are completely right. Fixed here: https://github.com/msaelices/mojo/commit/a034f7bf71fbf9fd99026d67d0093d34da55d2ae